core/llm/llms/OpenRouter.ts

```diff
@@ -0,0 +1,22 @@
+import { LLMOptions, ModelProvider } from "../../index.js";
+import OpenAI from "./OpenAI.js";
+
+class OpenRouter extends OpenAI {
+  static providerName: ModelProvider = "openrouter";
+  static defaultOptions: Partial<LLMOptions> = {
+    ...OpenAI.defaultOptions,
+    apiBase: "https://openrouter.ai/api/v1/",
+    model: "",
+    useLegacyCompletionsEndpoint: false,
+  };
+
+  protected _getHeaders() {
+    return {
+      ...super._getHeaders(),
+      "HTTP-Referer": "https://github.com/continuedev/continue",
```

Comment:
shouldn't this be pearai? since we added this

Reply:
I fixed it.